### PR TITLE
Docs: found 4 vulnerabilities (2 low, 2 high)

### PR DESCRIPTION
**Pull request by Luxcium**
"dependencies": {
    "glob": "^7.1.2", // Newer version of glob is available (7.1.6).(npm-outdated)
    "handlebars": "^4.0.10", // Newer version of handlebars is available (4.7.6).(npm-outdated)
    "handlebars-helpers": "^0.9.6", // Newer version of handlebars-helpers is available (0.10.0).(npm-outdated)
    "markdown-toc": "^1.1.0", // Newer version of markdown-toc is available (1.2.0).(npm-outdated)
    "parse-comments": "^1.0.0" // Updated from "^0.4.3" ― fixed 4 of 4 vulnerabilities in 257 scanned packages
  }

### CODE REVIEW

File: docs/package.json [hunk 1]
```diff
@@ -16,6 +16,6 @@
     "handlebars": "^4.0.10",
```
Comment (Luxcium) [inline docs/package.json:16]: A newer version of handlebars is available (4.7.6).(npm-outdated)

File: docs/package.json [hunk 2]
```diff
@@ -16,6 +16,6 @@
...
     "handlebars-helpers": "^0.9.6",
```
Comment (Luxcium) [inline docs/package.json:17]: A newer version of handlebars-helpers is available (0.10.0).(npm-outdated)

File: docs/package.json [hunk 3]
```diff
@@ -16,6 +16,6 @@
...
     "markdown-toc": "^1.1.0",
```
Comment (Luxcium) [inline docs/package.json:18]: A newer version of markdown-toc is available (1.2.0).(npm-outdated)

**Review by Luxcium — COMMENTED** [review]
"dependencies": {
    "glob": "^7.1.2", // Newer version of glob is available (7.1.6).(npm-outdated)
    "handlebars": "^4.0.10", // Newer version of handlebars is available (4.7.6).(npm-outdated)
    "handlebars-helpers": "^0.9.6", // Newer version of handlebars-helpers is available (0.10.0).(npm-outdated)
    "markdown-toc": "^1.1.0", // Newer version of markdown-toc is available (1.2.0).(npm-outdated)
    "parse-comments": "^1.0.0" // Updated from "^0.4.3" ― fixed 4 of 4 vulnerabilities in 257 scanned packages
  }

**Review by Luxcium — COMMENTED** [review]
When upgrading all to latest version: 

          === npm audit security report ===                        
                                                                                
┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ set-value                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=2.0.1 <3.0.0 || >=3.0.1                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ parse-comments                                               │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ parse-comments > map-schema > set-value                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1012                            │
└───────────────┴──────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ set-value                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=2.0.1 <3.0.0 || >=3.0.1                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ parse-comments                                               │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ parse-comments > map-schema > union-value > set-value        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1012                            │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 2 high severity vulnerabilities in 281 scanned packages
  2 vulnerabilities require manual review. See the full report for details.

**Comment by cpyarger** [comment]
Thank you

**Pull request merged**